### DISCUSSION TITLE
问一下大佬有没有红警2原版的修改器，只有尤里复仇的吗

### DISCUSSION THREAD

**Discussion by GleenJi** [AdjWang/RA2YurisRevengeTrainer#40] [Q&A]
如题

**Comment by AdjWang** [comment]
没做过，目前只有尤里复仇的。你要是用 CE 的话，可以从 https://www.cnblogs.com/viewll/p/4768880.html 找到红警2 原版的脚本，挺好用。

**Reply by GleenJi** [reply]
好的，感谢大佬的付出，目前用过的最好的红警2修改器
